src/ExtensionsSample/Program.cs

```diff
@@ -31,6 +31,7 @@ public static void Main(string[] args)
             config.UseFiles(filesConfig);
             config.UseTimers();
             config.UseSample();
+            config.UseMobileApps();
```

Comment:
I see that the ExtensionsSample project currently doesn't start, before my changes. So I'm unable to F5 and verify at this time.

Reply:
Looks like the sample has been broken since this commit: https://github.com/Azure/azure-webjobs-sdk-extensions/commit/f6ab149364c952ac2683fe1bd363cc09e61af269 where the custom Table binding I had was removed. @brettsam?

Reply:
It was removed b/c `ITableArgumentBinding` was removed and it didn't work anymore. I seem to remember you and @MikeStall telling me to delete this.

As for why it wasn't noticed -- I know I personally always remove everything from the TypeLocator other than the single extension I'm testing. Otherwise you get a bunch of noise that you need to work around. I'm assuming others must be doing the same thing or not many people are using this